packages/pigeon/platform_tests/alternate_language_test_plugin/ios/Classes/AlternateLanguageTestPlugin.m

```diff
@@ -258,6 +263,19 @@ - (void)callFlutterNoopWithCompletion:(void (^)(FlutterError *_Nullable))complet
   }];
 }
 
+- (void)callFlutterThrowErrorWithCompletion:(void (^)(id _Nullable,
+                                                      FlutterError *_Nullable))completion {
+  [self.flutterAPI throwErrorWithCompletion:^(id value, NSError *error) {
```

Comment:
These need to be FlutterError now, since I landed my PR first (to unblock CI work).